python/kwiver/sprokit/adapters/__init__.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
```

Comment:
Are we still supporting Python 2? (My guess is yes, but I just want to check)

Reply:
Yes, we are still supporting Python 2. Python 3 is the default, but users can switch to Python 2 in cmake.